Testreglar/2.2.2/Nett/nett-2.2.2a.json

```diff
@@ -0,0 +1,328 @@
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Innhald beveger seg, blinkar eller rullar, utan å bli vist samtidig med anna innhald."
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er bevegelsen, blinkinga eller rullinga del av ei handling som er avgjerande?",
+			"ht": "<p>Avgjerande (nødvendige) handlingar er typisk noko som skjer i sanntid, og kor det å pause/stoppe/skjule vil gjere handlinga ugyldig.</p>\n<p>Eksempel kan vere handel på børs, aksjon, klokke og liknande situasjonar der brukaren treng live-oppdateringar.</p>\n<p><strong>Merk:</strong> Merknad 4 til suksesskriteriet lyder slik: \"En animasjon som inngår i forhåndsinnlasting eller lignende, kan anses som avgjørende dersom det ikke kan forekomme interaksjon for noen brukere i denne fasen, og dersom mangel på en framdriftsindikator kunne forvirre brukerne og skape inntrykk av at innholdet er enten frosset eller ødelagt.\"</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Innhald som beveger seg, blinkar eller rullar er del av ei handling som er avgjerande."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Finst det ein mekanisme på testsida, for å pause, stoppa eller skjula innhaldet som beveger seg, blinkar eller rullar? ",
+			"ht": "<p>Mekanismen kan for eksempel vere ein stopp- eller pauseknapp.</p>\n<p><strong>Merk:</strong> Det er ikkje tilstrekkeleg at innhaldet pauser/stoppar berre når det er i fokus. </p>",
+			"type": "jaNei",
+			"kilde": [
+				"G4",
+				"G186",
+				"G187"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				}
+			}
+		},
+		{
+			"stegnr": "3.6",
+			"spm": "Ligg mekanismen i direkte tilknytning til innhaldet som beveger seg, blinkar, rullar?",
+			"ht": "<p>Plasseringa kan for eksempel vere rett over eller rett ved sida av innhaldet.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G186"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Ligg mekanismen på starten av testsida?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjekk om mekanismen er plassert nær starten av testsida.</p>\n<p><strong>Merk:</strong> Du skal ikkje måtte scrolle vekk frå starten av sida for å nå mekanismen.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G186"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				}
+			}
+		},
+		{
+			"stegnr": "3.8",
```

Comment:
Samsvar mellom spørsmål og svaralternativ. For eksempel: "Vert innhaldet sett på pause, stoppa eller skjult når mekanismen aktiverast?